plugins/ImageGraph/StaticGraph/GridGraph.php

```diff
@@ -372,7 +372,7 @@ protected function getGridTopMargin($horizontalGraph, $verticalLegend)
     private function getHorizontalLegendHeight()
     {
         list($maxMetricLegendWidth, $maxMetricLegendHeight) =
-            $this->getMaximumTextWidthHeight(array_values($this->ordinateLabels), $this->legendFontSize);
+            $this->getMaximumTextWidthHeight(array_values($this->ordinateLabels));
```

Comment:
`getMaximumTextWidthHeight` does only have one parameter.

Reply:
But wouldn't it be more correct to respect the parameter in `getMaximumTExtWidthHeight`?

If the value is passed through to the nested `getTextWidthHeight()` call, then `$maxMetricLegendHeight` will change from `9` to `11`, using the actual legend font size instead of the regular font size :thinking: 

Don't know what needs to be done to see a visual difference, though.

Reply:
I'll try that and see if that has any impact on our UI tests.

Reply:
@mneudert Based on https://builds-artifacts.matomo.org/matomo-org/matomo/fixunusedcode/13517196878/ImageGraph_horizontal_bar.png I guess I would rather remove it again, as it seems to increase the spacing unnecessarily. 🤔